README.md

```diff
@@ -171,7 +171,8 @@ A sample configuration file with all options is available [here](https://github.
 * `member-ordering` enforces member ordering. Rule options:
     * `public-before-private` All public members must be declared before private members.
     * `static-before-instance` All static members must be declared before instance members.
-    * `variables-before-functions` All variables needs to be declared before functions.
+    * `variables-before-functions` All member variables need to be declared before member functions.
+       Member variables initialized to a function literal are treated as member functions.
```

Comment:
Eh, it's not perfect because functions assigned with `=` are properties really but we treat them as methods. (I don't think you'd call functions on the prototype "properties" but not sure.) Let me think about this, but I'm inclined just to leave it as it is because I don't see a big clarity benefit worth making a breaking change